collector.go

```diff
@@ -220,6 +228,26 @@ func getDatabaseIndexes(dbStats *dbStats) float64 {
 	return value
 }
 
+func countDatabaseTasksByStatus(data []byte, status string) int {
+	count := 0
+	jp.ArrayEach(data, func(value []byte, dataType jp.ValueType, offset int, err error) {
+		if taskConnectionStatus, _ := jp.GetString(value, "TaskConnectionStatus"); taskConnectionStatus == status {
+			count++
+		}
+	}, "OngoingTasksList")
+
+	return count
+}
+
+func getDatabaseInactiveTasks(dbStats *dbStats) float64 {
+	return float64(countDatabaseTasksByStatus(dbStats.tasks, "NotActive"))
```

Comment:
Are `Active` and `NotActive` the only possible statuses? 

What do you think about an alternative approach where there is only single new metric `database_tasks` with labels `database`, `type`, and `connection_status`. This would allow for more flexibility when querying the metrics and also ensures possibility to add other metrics in the future.

Of course this will require more changes to the code to allow additional labels for per-database metrics.

Reply:
`Active` and `NotActive` don't seem to be the only options, there is [several](https://github.com/ravendb/ravendb/blob/5ebd70e9e23ff9d14f7de9833bb136250a04d77d/src/Raven.Client/Documents/Operations/OngoingTasks/GetTaskInfoResult.cs#L45) 

Adding extra labels would definitely open up more options and remove the need to know the status beforehand. I would also include a task name label as a unique identifier so that there would be no need for any count logic and rather use PromQL sum if one desires. What do you think?

Reply:
Thanks for these changes, it looks good now. Only the `name` label is something I'm unsure about. 

Name identifies a specific task. On one hand this gives detailed information about tasks. On the other, there exists only one combination of name and status for given task, and this combination will have value of 1, other statuses will be 0. Apart from this being a bit weird, it also leads to high cardinality of the metric, which might be problematic in some scenarios, e.g. when you pay per stored datapoint.

I'm not sure what the best course of action would be in this case, let me know what you think.

Reply:
Initially I had the idea of having only "1" as the metric value which provide a representation of all existing tasks regardless of state (state can also be added to as a label - apart from connection_status). But I agree it does already expose details and produce a high cardinality metric.

In that case, I would stick to the original approach (remove the name label, and aggregate tasks with same type and connection_status). (https://github.com/marcinbudny/ravendb_exporter/pull/6/commits/5709edffd415961c4bc7ffc4c385a5e021c96d47)